### PR TITLE
Move see all review link under the ratings.

### PR DESCRIPTION
**Pull request by StevenDufresne**
I find the "see all reviews" link distracting as it's the only right-aligned link that is next to the title in the sidebar.

In this PR I proposed we move it below the reviews which also seems to make the tabbing more logical where you tab through some reviews to reach the "all reviews" link. I wasn't sure which order makes more sense; "add my review" first or "see all reviews" first. I'm open to suggestions on that.

If this change goes in, I'll update the plugin directory to match.

### Screenshots

| Before | After |
|--------|-------|
| Screenshot 2024-06-05 at 10 51 01 AM  | Screenshot 2024-06-05 at 10 50 44 AM |

Note: The "after" screenshot data and star ratings don't match as I copied/pasted DOM parts from the live site since my local environment didn't have the ratings table.

### CODE REVIEW

**Comment by ryelle** [comment]
IMO having both links side by side looks very repetitive, of the two options I prefer the top right-aligned link.  Maybe just move the "see all" link under the heading?

Screenshot 2024-06-05 at 10 20 53 AM

**Comment by jasmussen** [comment]
No strong opinions here, happy to try what you feel the strongest for. One suggestion, though, could be this:
Group 1

Note the reduced verbiage.

**Comment by fcoveram** [comment]
I like @jasmussen's idea, and I would increase the spacing between the two links and the bars area to make the difference more clear.

**Comment by StevenDufresne** [comment]
Update layout in [40691ee](https://github.com/WordPress/wporg-theme-directory/pull/111/commits/40691ee40785f3f4e76a97468cf461f2c6bd628b).

| Mobile | Desktop |
|--------|--------|
| Screenshot 2024-06-12 at 11 07 24 AM | Screenshot 2024-06-12 at 11 07 31 AM |

**Comment by jasmussen** [comment]
Works for me!

File: source/wp-content/themes/wporg-themes-2024/inc/block-bindings.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ function get_meta_block_value( $args, $block ) {
 			return sprintf(
 				'<a href="%s">%s</a>',
 				esc_url( get_support_url( $theme->slug . '/reviews/' ) ),
-				__( 'See all reviews', 'wporg-themes' )
+				__( 'See all<span class="screen-reader-text"> reviews</span>', 'wporg-themes' )
```
Comment (ryelle) [inline source/wp-content/themes/wporg-themes-2024/inc/block-bindings.php:81]: I updated this to bring back "reviews" for screen reader users, so navigating by link isn't a mystery :)

**Review by ryelle — APPROVED** [review]
Works for me 👍🏻

**Pull request merged**